Sources/AppStoreConnectCLI/Model/BetaGroup.swift

```diff
@@ -62,6 +65,37 @@ extension BetaGroup {
     }
 }
 
+extension BetaGroup: Equatable, Hashable {
+
+    static func == (lhs: BetaGroup, rhs: BetaGroup) -> Bool {
+        return lhs.id == rhs.id &&
+            lhs.groupName == rhs.groupName &&
+            lhs.publicLinkEnabled == rhs.publicLinkEnabled &&
+            lhs.publicLinkLimit == rhs.publicLinkLimit &&
+            lhs.publicLinkLimitEnabled == rhs.publicLinkLimitEnabled
+    }
+
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(id)
+        hasher.combine(groupName)
+        hasher.combine(publicLinkEnabled)
+        hasher.combine(publicLinkLimit)
+        hasher.combine(publicLinkLimitEnabled)
+    }
```

Comment:
We could potentially only use the id for matching.. do we ever foresee having duplicates?

Did you have to override the default implementations because of the testers file path? There may also be a different solution here 🤔

Reply:
Yeh, for now, this is only for comparing the BetaGroup object difference, not for testers. when doing something like `Set(localGroups).subtracting(serverGroups)` I can get the difference and update the server by local